docs/source/docs_user/docs_user_data_catalogue.rst

```diff
@@ -9,1361 +9,35 @@ The CABD data catalogue contains the the information you need to make sense of t
- **Definition:** *The reason the fishway was designed and implemented.* 
- 
- **Field name:** purpose
-
-|dcfishreturn|
-
-.. _rivnameen:
-
-River/Stream Name (English)
-+++++++++++++++++++++++++++
- **Definition:** *Name of river/stream in which the feature is recorded (English).* 
- 
- **Field name:** river_name_en
-
-|dcfishreturn|
-
-.. _rivnamefr:
-
-River/Stream Name (French)
-++++++++++++++++++++++++++
- **Definition:** *Name of river/stream in which the feature is recorded (French).* 
- 
- **Field name:** river_name_fr
-
-|dcfishreturn|
-
-.. _knownnotuse:
-
-Species Known to Not Use
-++++++++++++++++++++++++
- **Definition:** *Species where it is known that the fishway presents a significant barrier to migration.*
- 
- **Field name:** known_to_not_use
-
-|dcfishreturn|
-
-.. _knowntouse:
-
-Species Known to Use
-++++++++++++++++++++
- **Definition:** *Species that are known to use the fishway.* 
-
- **Field name:** known_to_use
- 
-|dcfishreturn|
-
-.. _strucnameen:
-
-Structure Name (English)
-++++++++++++++++++++++++
- **Definition:** *The given or known name of the fishway structure or the dam with which it is associated (English).* 
- 
- **Field name:** structure_name_en
-
-|dcfishreturn|
-
-.. _strucnamefr:
-
-Structure Name (French)
-+++++++++++++++++++++++
- **Definition:** *The given or known name of the fishway structure or the dam it is associated with (French).* 
- 
- **Field name:** structure_name_fr
-
-|dcfishreturn|
-
-.. _systemid:
-
-System Identifier
-+++++++++++++++++
- **Definition:** *Unique identifier for each fishway point.* 
- 
- **Field name:** cabd_id
-
-|dcfishreturn|
-
-.. _success:
-
-Transit Success Estimate (%)
-++++++++++++++++++++++++++++
- **Definition:** *Estimated percentage of individuals that successfully pass through the fishway.*
- 
- **Field name:** estimate_of_passage_success_pct
-
-|dcfishreturn|
-
-.. _yearconst:
-
-Year Constructed
-++++++++++++++++
- **Definition:** *Year in which the fishway structure was built.* 
- 
- **Field name:** year_constructed
- 
-|dcfishreturn|
+    <script>
+    function loadDataSources() {
+    
+      let url = "https://cabd-web.azurewebsites.net/cabd-api/docs?options=types"
+      let xhr = new XMLHttpRequest();
+      // Making our connection 
+       xhr.open("GET", url, true);
+   
+      // function execute after request is successful
+      xhr.onreadystatechange = function () {
+        if (this.readyState == 4 && this.status == 200) {
+            let element = document.getElementById("datasourcecontent");
+            element.innerHTML = this.responseText;      
+            
+            //make a data table out of the nhn watershed id field
+            //across all feature types
+            $("table[id^='datatable_attribute_nhn_watershed_id']").each(function(){
+               $("#" + this.id).DataTable();
+            });            
+        }
+      }
+      // Sending our request
+      xhr.send();
+    }
+    loadDataSources();
+    </script>
+  
```

Comment:
Add the following to the bottom of the page:

.. toctree::
   :hidden:

   docs_user_data_sources/docs_user_data_sources_csv_download.rst